add-matcha.php

```diff
@@ -0,0 +1,56 @@
+<?php
+//connect to the database and hide the db-connection.php into gitignore
+//get products from the database by creating a separate file for functions
+require_once 'db-connection.php';
+require_once 'functions.php';
+$db = getDb();
+
+if(isset($_POST['submit']))
```

Comment:
Need to add name="submit" to submit input for this to be set when submitting form

Reply:
I will make this change in the next task!  I forgot to take them out in my experiment file!